### PR TITLE
Enable clang-tidy in aten/src/ATen/native/nested/

### PR DESCRIPTION
**Pull request by cyyever**
Fixes #ISSUE_NUMBER

### CODE REVIEW

**Review by Skylion007 — APPROVED** [review]

File: aten/src/ATen/native/nested/NestedTensorUtils.h [hunk 1]
```diff
@@ -223,11 +223,12 @@ struct NestedNode {
       : _is_leaf(false), _children(std::move(children)) {}
   explicit NestedNode(TensorList children)
       : _is_leaf(false), _children(children.vec()) {}
-  // NestedNode(NestedNode&) = delete;
-  // NestedNode(const NestedNode&) = delete;
-  // NestedNode& operator=(NestedNode) = delete;
   explicit NestedNode(T payload)
       : _is_leaf(true), _payload(std::move(payload)) {}
+  NestedNode(const NestedNode&) = delete;
+  NestedNode& operator=(const NestedNode&) = delete;
```
Comment (Skylion007) [inline aten/src/ATen/native/nested/NestedTensorUtils.h:229]: Any indication why these were commented out?

Comment (cyyever) [reply aten/src/ATen/native/nested/NestedTensorUtils.h:229]: I don't know.

**Pull request closed without merging**